### PR TITLE
[14.0][MIG] stock_available_immediately

### PR DESCRIPTION
**Pull request by florian-dacosta**
Standard migration (just fix tests)

Depends on 

- [ ] https://github.com/OCA/stock-logistics-warehouse/pull/1037

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]
LGTM :+1: 
Code review

File: stock_available_immediately/models/product_product.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from odoo import api, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _compute_available_quantities_dict(self):
+        res, stock_dict = super(
+            ProductProduct, self
+        )._compute_available_quantities_dict()
```
Comment (victoralmau) [inline stock_available_immediately/models/product_product.py]:
```suggestion
        res, stock_dict = super()._compute_available_quantities_dict()
```

File: stock_available_immediately/models/product_product.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        for product in self:
+            res[product.id]["immediately_usable_qty"] -= stock_dict[product.id][
+                "incoming_qty"
+            ]
+        return res, stock_dict
+
+    @api.depends("virtual_available", "incoming_qty")
+    def _compute_available_quantities(self):
+        return super(ProductProduct, self)._compute_available_quantities()
```
Comment (victoralmau) [inline stock_available_immediately/models/product_product.py]:
```suggestion
        return super()._compute_available_quantities()
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Review by victoralmau — APPROVED** [review]

**Pull request merged**